backend/src/infrastructure/database/repositories/steam.apps.repository.js

```diff
@@ -67,4 +67,16 @@ export class SteamAppsRepository {
 
     return SteamApp.manyFromDbEntries(response);
   }
+
+  async getSteamAppsById(games) {
```

Comment:
Title of function does not match the argument. Better provide it an array of ids, call it like this then:
`getSteamAppsById(games.map(game => game.id))`